source/_includes/asides/getting_started_navigation.html

```diff
@@ -67,7 +67,7 @@ <h1 class="title delta">Getting Started Guide</h1>
       </li>
       <li>{% active_link {{site_root}}/topics/ Advanced Topics %}</li>
       <li>{% active_link {{site_root}}/details/ Details %}</li>
-      <li>{% active_link /getting-started/android/ Add to Android Homescreen %}</li>
+      <li>{% active_link /getting-started/android/ Add to Android/iOS Homescreen %}</li>
```

Comment:
This is a bit misleading since the page it's linking to is specifically called Android.